src/rest-client.ts

```diff
@@ -426,12 +427,16 @@ export class RestClient extends BaseRestClient {
   getFeeRates(
```

Comment:
Can you make a `getFeeRatesV2` method that uses an object as a param instead of these individual params? Please also add a deprecated note on this method, similar to what we did in other SDKs to help transition to the object-param endpoint (so we can replace getFeeRates with an object fn instead in future).

Reply:
Done